packages/block-editor/src/hooks/style.js

```diff
@@ -127,6 +129,7 @@ const skipSerializationPathsEdit = {
 const skipSerializationPathsSave = {
 	...skipSerializationPathsEdit,
 	[ `${ SPACING_SUPPORT_KEY }` ]: [ 'spacing.blockGap' ],
+	[ `${ MEDIA_SUPPORT_KEY }` ]: [ MEDIA_SUPPORT_KEY ],
```

Comment:
This isn't quite working as intended yet — I notice the background styles are being serialized. I'll fix this up next week so that they're skipped for these styles.

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/53934/commits/546c0ad1073fd61d7b317abf994400f4953a8d31